SH4RE/SH4RE/Account/FavouritesModel.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  FavouritesModel.swift
+//  SH4RE
+//
+//  Created by Americo on 2023-03-17.
+//
+
+import SwiftUI
+
+class FavouritesModel: ObservableObject {
+    @Published var favourites:Set<String> = []
+    
+    func addFavourite(listingID:String){
+        favourites.insert(listingID)
```

Comment:
we should push to favourites table in firebase here. Can you add a todo comment for @bbryanb333

Reply:
np